twitter-borrowbot/services.py

```diff
@@ -2,7 +2,7 @@
 import re
 import requests
 import datetime
-
+from twitterbotErrors import *
```

Comment:
Please avoid wildcard imports as discussed in PEP8.  They make NameErrors difficult to find.

Reply:
Do you think I should `from twitterbotErrors import thing1, thing 2 ...` OR just `import twitterbotErrors as errors`

Reply:
`from twitterbotErrors import thing1, thing2 ...` makes fewer changes on the other lines which makes the PR easier to review.